LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -232,6 +232,13 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                     break;
                 case var messageType and (LnsMessageType.DownlinkMessage or LnsMessageType.RouterConfig):
                     throw new NotSupportedException($"'{messageType}' is not a valid message type for this endpoint and is only valid for 'downstream' messages.");
+                case var messageType and (LnsMessageType.ProprietaryDataFrame
+                                          or LnsMessageType.MulticastSchedule
+                                          or LnsMessageType.TimeSync
+                                          or LnsMessageType.RunCommand
+                                          or LnsMessageType.RemoteShell):
+                    this.logger.LogWarning($"'{messageType}' ({messageType.ToBasicStationString()}) is not handled in current LoRaWan Network Server implementation.");
+                    break;
```

Comment:
Do we want a unit test to cover this (the fact that we tolerate these types even though we don't handle them)?